src/styles/components/_sign-up-success-form.scss

```
@@ -0,0 +1,24 @@
+@import '../abstracts/_abstracts-dir';
+
+.sign-up-success-form {
+  display: flex;
+  flex-direction: column;
+  background-color: $background;
+
+  &__header {
+    padding: rem(16px 0);
+    margin-bottom: rem(24px);
+
+    @include flex-center();
```

Is there any difference between this?
```suggestion
    display: flex;
    justify-content: center;
```